packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+# Number Field Updates (IxD)
+
+## Overview
+
+### Background
+
+- [Bring number-column formatting options to nimble-number-field](https://github.com/ni/nimble/issues/1855)
+- [Number IxD (Figma)](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1603-3464)
+
+The `nimble-number-field` leverages
+[fast-foundation](https://github.com/ni/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/number-field/number-field.ts)
+for its behavior, so we've inherited much of the behavior, including max/min
+validation and localization.
+
+For supported languages (English, French, German, Japanese, Chinese), the
+_decimal separator_ does not align with user expectations in German. For
+example, `1000000.01`
+[should](https://randombits.dev/articles/number-localization/locale-list) render
+as `1000000,01` in German but currently renders as `1000000.01` across all
+locales.
+
+Additionally, clients have requested the ability to define max/min values for
+the `nimble-number-field` to support form validation without automatically
+coercing user-entered values to fit within the specified range.
+
+In comparison, the HTML `<input type="number">` natively supports locale-aware
+_decimal separators_ and does not coerce values to the min/max limit.
+
+This IxD proposes updates to the Number Field to natively support localization,
+add coercion options, and define decimal precision.
+
+## Usage
+
+Number fields are used in three distinct ways:
+
+1. **With Increment/Decrement Buttons (Steppers)**
+
+    - Suitable for small, controlled ranges of values where users adjust numbers
+      incrementally (e.g., specifying a slot number).
+    - Discrete integers with a predictable range.
```

Comment:
I'm not sure we need to be so opinionated as to assume using the step buttons only is useful for integers.

Reply:
Not only integers, but a small, predictable range of integers!

Today it's not possible to make clear to a user what the step value is, nor is it possible to contrain the step increments, so I'm confident more users fail than are helped by an increment/decrement accelerator on decimal numbers. E.g., 99.25 + increment = 100.25 (not helpful)

Regardless, the existing number-text-field handles the small integer case well, so I'm specifically excluding steppers from these requirements.